src/devices/Vl53L1X/Roi.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Vl53L1X
+{
+    /// <summary>
+    /// Holds the ROI information of the device
+    /// </summary>
+    public class Roi
+    {
+        /// <summary>
+        /// Width of the ROI
+        /// </summary>
+        public ushort Width { get; }
```

Comment:
would it make sense to have this as a length as well?

Reply:
No. This is not a range value. It is the amount of active SPADS in the SPAD array of the device. I have added further explanations in the comment of the ROI property of the VL53l1X class and added range checks for the parameter.